operations/operations/publishing/publishing-guidelines/documentation-style-guide.md

```diff
@@ -23,38 +19,37 @@ The title appears in the left-hand navigation and at the top of the table of con
 
 ### Document sections
 
-To make pages easier for people to quickly scan for the content that they're looking for, break your 
-document up into logical sections. Each section should have a heading, and the heading should relate 
-to the content of the section. A section heading is not required if you have only one section.
+To make pages easier for people to quickly scan for the content that they're looking for, break your document up into logical sections. Each section should have a heading, and the heading should relate to the content of the section. A section heading is not required if you have only one section.
 
 **Use sentence case for the section heading (e.g. "This section is about types of headings") except where a word/phrase is a proper noun (e.g. "Mattermost"), the name of a country/region (e.g. "The United States of America"), and so on.**
 
-
 ### Abstract
 
-The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. 
-It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the 
-description is displayed in the search results along with the page title. People read the description to 
-help them decide if the document is the one that they want.
+The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the description is displayed in the search results along with the page title. People read the description to help them decide if the document is the one that they want. 
+
+An exception is Customer Education content that has no need to outline the content but set more specific expectations for a well-defined audience. 
+
+Clearly name the audience as it can be narrow (System Administrators), but also large (System Administrators, Project Managers, and IT Leaders in general). When setting the expectations answer these questions
+
+- How will the reader benefits from reading the guide - what exact knowledge is taught?
+- How long will it take to work through the guide?
+- What similar topics are not covered?
```

Comment:
Does this mean those topics are covered in the guide or should be mentioned as not covered in this specific guide?

Reply:
What similar topics are not covered in this guide (the guide you are writing)?

The Idea is to set the expectations as clear as possible. For example writing about plug-ins it might be a good idea to make sure it's not about extensions.

Reply:
That's a good explanation. One option is to add it to the final point for clarity.

Reply:
Sure thing! I'll change it from

> What similar topics are not covered?

to 

> What similar topics are not covered in this guide?

Reply:
"Are there similar topics not covered in this guide?" could also be an option?